### ISSUE TITLE
Disable network debug information when attaching to terminal

### ISSUE THREAD

**Issue by gionapaolini** [Azure/vscode-kubernetes-tools#900]
It often happens (not all the time) that when opening a terminal on a pod from the extensions I get network debug information. 
It is the exact same problem mentioned here https://stackoverflow.com/questions/43144822/disabling-network-logs-on-kubernetes-when-running-kubectl-exec

The proposed solution works if I indeed attach manually with the command `kubectl exec -it <PODNAME> -- /bin/bash `, but it doesn't for the extensions. So it seems that the extensions randomly set DEBUG=true.

**Comment by itowlson** [comment]
I don't think we set `DEBUG=true` anywhere. Not sure why you would be seeing this from the extension but not from your CLI.  What happens if you run the command from the VS Code embedded terminal?

**Comment by gionapaolini** [comment]
Is it supposed to use the default integrated terminal under the hood?
Because I just checked and integrated terminal also work fine, but when I tried with the extensions, I again saw the network information

**Comment by bibic-vegait** [comment]
I have the same issue that started happening one or two weeks ago, after the latest upgrade (Version: 1.73.1). When using `kubectl exec` command in the terminal, I do not have this issue, only when using VS Code plugin to run Terminal. The annoying thing is that the Terminal inside the pod is useless if I open it using the plugin, as the amount of debug outputs is overwhelming. Here is the snippet of the logs that I find after executing the pod (this is what I see, this is not the formatting issue!):

`I1122 11:19:16.368106   34391 log.go:195] (0xc0001f6000) (0xc00051a140) Create stream
                                                                                     I1122 11:19:16.368763   34391 log.go:195] (0xc0001f6000) (0xc00051a140) Stream added, broadcasting: 1
                                                              I1122 11:19:16.799343   34391 log.go:195] (0xc0001f6000) Reply frame received for 1
                     I1122 11:19:16.800157   34391 log.go:195] (0xc0001f6000) (0xc00051a1e0) Create stream
                                                                                                          I1122 11:19:16.800183   34391 log.go:195] (0xc0001f6000) (0xc00051a1e0) Stream added, broadcasting: 3
                                                                                   I1122 11:19:17.233067   34391 log.go:195] (0xc0001f6000) Reply frame received for 3
                                          I1122 11:19:17.233369   34391 log.go:195] (0xc0001f6000) (0xc00063a1e0) Create stream
   I1122 11:19:17.233392   34391 log.go:195] (0xc0001f6000) (0xc00063a1e0) Stream added, broadcasting: 5
                                                                                                        I1122 11:19:17.549091   34391 log.go:195] (0xc0001f6000) Reply frame received for 5
                                                               I1122 11:19:17.549340   34391 log.go:195] (0xc0001f6000) (0xc00051a280) Create stream
                        I1122 11:19:17.549355   34391 log.go:195] (0xc0001f6000) (0xc00051a280) Stream added, broadcasting: 7
 I1122 11:19:17.982095   34391 log.go:195] (0xc0001f6000) Data frame received for 5
                                                                                   I1122 11:19:17.982163   34391 log.go:195] (0xc00063a1e0) (5) Data frame handling
                                       I1122 11:19:17.982183   34391 log.go:195] (0xc0001f6000) Data frame received for 5
                                                                                                                         I1122 11:19:17.982216   34391 log.go:195] (0xc00063a1e0) (5) Data frame handling
                                                                             I1122 11:19:17.982684   34391 log.go:195] (0xc0001f6000) Reply frame received for 7
                                    / # I1122 11:19:17.983612   34391 log.go:195] (0xc00063a1e0) (5) Data frame sent
                                                                                                                    I1122 11:19:17.984800   34391 log.go:195] (0xc00051a280) (7) Writing data frame
                                                                       I1122 11:19:18.003873   34391 log.go:195] (0xc00051a1e0) (3) Writing data frame
                          I1122 11:19:18.356689   34391 log.go:195] (0xc0001f6000) Data frame received for 5
                                                                                                            I1122 11:19:18.356755   34391 log.go:195] (0xc00063a1e0) (5) Data frame handling
                                                                I1122 11:19:18.356784   34391 log.go:195] (0xc00063a1e0) (5) Data frame sent
/ # I1122 11:19:22.244217   34391 log.go:195] (0xc00051a1e0) (3) Writing data frame
                                                                                   I1122 11:19:22.574507   34391 log.go:195] (0xc0001f6000) Data frame received for 5
                                         I1122 11:19:22.574570   34391 log.go:195] (0xc00063a1e0) (5) Data frame handling
                                                                                                                         I1122 11:19:22.574597   34391 log.go:195] (0xc00063a1e0) (5) Data frame sent

/ #`

**Comment by icaria555** [comment]
I also got the same issue, currently I am using Version: 1.93.0 (Universal) of vscode and Version: v1.3.16
 of Kubernetes plugin
